buildSrc/src/main/java/org/elasticsearch/gradle/PublishPlugin.java

```diff
@@ -37,17 +39,38 @@
 import static org.elasticsearch.gradle.util.GradleUtils.maybeConfigure;
 
 public class PublishPlugin implements Plugin<Project> {
+    static final String MAVEN_SHADOW = "publish.maven.shadow";
+    static final String MAVEN_JAR = "publish.maven.jar";
 
     @Override
     public void apply(Project project) {
         project.getPluginManager().apply(BasePlugin.class);
-        project.getPluginManager().apply("nebula.maven-nebula-publish");
+        project.getPlugins().apply(MavenPublishPlugin.class);
```

Comment:
Shoudl we use the `PluginManager` version consistently here?

Reply:
fixed